@here/harp-webpack-utils/scripts/HarpWebpackConfig.js

```diff
@@ -35,18 +51,20 @@ export function addHarpWebpackConfig(config?: Configuration, harpConfig?: HarpWe
         },
         mode
     };
-    const typescriptConfig: Configuration = {
+    const typescriptConfig /*: Configuration*/ = {
         resolve: {
             extensions: [".web.ts", ".ts", ".tsx"]
         },
         module: {
-            rules: [{ test: /\.tsx?$/, loader: "ts-loader" }]
+            rules:
+                userConfig.module === undefined || userConfig.module.rules === undefined
+                    ? [{ test: /\.tsx?$/, loader: "ts-loader" }]
+                    : []
         }
     };
-    const mainConfig =
-        mainEntry !== undefined && mainEntry.endsWith(".ts")
-            ? WebpackMerge.smart(baseConfig, typescriptConfig)
-            : baseConfig;
+    const mainConfig = isTypescript(mainEntry)
+        ? WebpackMerge.smart(baseConfig, typescriptConfig)
+        : baseConfig;
     const bundles = [
         WebpackMerge.smart(
             {
```

Comment:
General comment.
`stats` is also very specific to application. We're overwriting too much of `webpack`s defaults.

I think only "exclude: /resources/" should remain.